completion_aggregator/urls.py

```diff
@@ -9,5 +9,6 @@
 
 urlpatterns = [
     url(r'^v1/', include('completion_aggregator.api.v1.urls', namespace='completion_api_v1')),
+    url(r'^v0/', include('completion_aggregator.api.v0.urls', namespace='completion_api_v0')),
```

Comment:
@jcdyer shall we update edx-platform to use this url instead? or shall we keep the same url there /api/completion/v0 ?

Reply:
edx-platform will need to use /api/completion/v0/, because we need this to support existing clients (mobile apps) that are out in the wild and cannot be modified.

Reply:
@jcdyer could you give me access to push a branch to edx-platform? I ahve the code ready